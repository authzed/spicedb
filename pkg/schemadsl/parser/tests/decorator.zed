definition mydefinition {
    @example
    relation foo: sometype
}