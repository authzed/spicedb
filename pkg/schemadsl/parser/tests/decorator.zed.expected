NodeTypeFile
  end-rune = 66
  input-source = decorator test
  start-rune = 0
  child-node =>
    NodeTypeDefinition
      definition-name = mydefinition
      end-rune = 66
      input-source = decorator test
      start-rune = 0
      child-node =>
        NodeTypeRelation
          end-rune = 64
          input-source = decorator test
          relation-name = foo
          start-rune = 43
          allowed-types =>
            NodeTypeTypeReference
              end-rune = 64
              input-source = decorator test
              start-rune = 57
              type-ref-type =>
                NodeTypeSpecificTypeReference
                  end-rune = 64
                  input-source = decorator test
                  start-rune = 57
                  type-name = sometype
          decorator-node =>
            NodeTypeDecorator
              decorator-name = example
              end-rune = 37
              input-source = decorator test
              start-rune = 30